src/components/Comment.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
```

Comment:
functional component can be used (there is no state)

Reply:
Done